e2e-tests/cypress/e2e/journeys/insurance/account/sign-in/enter-code/validation/account-sign-in-enter-code-validation.spec.js

```diff
@@ -69,7 +70,7 @@ context('Insurance - Account - Sign in - Enter code - validation', () => {
     });
 
     describe('when submitting a valid security code', () => {
-      it(`should redirect to ${DASHBOARD}`, () => {
+      it(`should redirect to ${ROOT}${DASHBOARD}`, () => {
```

Comment:
what if the dashboard in routes had DASHBOARD: ${ROOT}${DASHBOARD} then would not need to add it here?

Reply:
Great shout, this was the case in UI constants, but E2E was missed in the update. Fixed